From: Niko Tyni <ntyni@debian.org>
Date: Sat, 29 Jul 2017 23:32:51 +0300
Subject: Also lower the optimization level of op.c and opmini.c on sh4

This amends
 debian/hppa_op_optimize_workaround.diff
 debian/hppa_opmini_optimize_workaround.diff

Bug-Debian: https://bugs.debian.org/869373
---
 cflags.SH | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cflags.SH b/cflags.SH
index 02a5e71..6cb3cae 100755
--- a/cflags.SH
+++ b/cflags.SH
@@ -473,9 +473,9 @@ for file do
     # or customize here
 
     case "$file" in
-    op|opmini) : work around http://bugs.debian.org/838613 and 869122
+    op|opmini) : work around http://bugs.debian.org/838613 869122 869373
         case $archname in
-        hppa-*)
+        hppa-*|sh4-*)
             optimize="$optimize -O0";;
         esac;;
 
